salt/INSTALL.md

```diff
@@ -0,0 +1,108 @@
+# Deploying OpenSDS with Salt
+
+Deploy OpenSDS (www.opensds.io) using Salt. This is an experimental solution using repeatable patterns of jinja/yaml to deploy cloud-native stack using infrastructure as code. Please use Salt 2018.3.4 or earlier while potential impacts from the <a href="https://docs.saltstack.com/en/develop/topics/releases/2019.2.0.html#non-backward-compatible-change-to-yaml-renderer">NON-BACKWARD-COMPATIBLE-CHANGE-TO-YAML-RENDERER</a> are reviewed by the <a href="https://github.com/saltstack-formulas">saltstack-formulas community</a>.
+
+Software versions
+=================
+Verified on CENTOS-7, UBUNTU-18, and OPENSUSE-15 with Salt version 2018.3.
+
+Solution View
+=============
+
+<a href="https://github.com/opensds/opensds">![Solution overview](solutionDesign.png)</a>
+
+Reference Vagrant setup
+=======================
+Refer to HACKING.md. Prefer a minimum of 6GB+4CPU per virutalized hosts.
+
+
+Procedure
+===========
+Deploy OpenSDS using the steps below. The expected installer duration is 20-55 minutes depending on your network bandwith and compute resources (we recommend at least 4CPU x 8G ram x 60G rootdisk).
+
+```
+ sudo -s
+ cd /root && git clone https://github.com/opensds/opensds-installer.git
+ cd opensds-installer/salt
+```
+
+Install Salt on UBUNTU/CENTOS/OpenSUSE_15
+```
+ ./install.sh -i salt
+```
+Reboot if kernel got upgraded. If in doubt, reboot anyway.
+```
+init 6
+```
+
+Review site deployment data to double check ipv4 adddres. Set "auth_strategy: noauth" if not using keystone.
+```
+ ip addr
+ vi site.j2
+```
+
+Deploy OpenSDS on UBUNTU/CENTOS/OpenSUSE_15
+```
+sudo -s
+cd /root/opensds-installer/salt/; install.sh -i opensds
```

Comment:
Please  change to `cd /root/opensds-installer/salt/; ./install.sh -i opensds`

Reply:
Done. thank you for noticing this